src/components/Cards/style.ts

```diff
@@ -39,7 +39,8 @@ const useStyles = makeStyles(() => ({
   tooltipArrow: {
     color: colors.gray.mid,
     borderColor: colors.gray.mid,
-    width: 13
+    width: 13,
+    marginBlock: '-0.67em !important'
```

Comment:
I'm not sure if this line is needed

Reply:
to było mi potrzebne, by nadpisać style siedzące głęboko wewnątrz Material-UI